webapp/client/src/pages/LoginPage.js

```diff
@@ -1,9 +1,19 @@
-import styled from "styled-components";
+import PropTypes from "prop-types";
+import StepHeaderButtons from "../components/StepHeaderButtons";
 
-const Placeholder = styled.div`
-  display: none;
-`;
-
-export default function LoginPage() {
-  return <Placeholder>Page content will go here</Placeholder>;
+export default function LoginPage({ prevUrl, backLinkText }) {
+  return (
+    <StepHeaderButtons backLinkUrl={prevUrl} backLinkText={backLinkText} />
+  );
 }
```

Comment:
Am I still missing something or are we showing the Placeholder now?

Reply:
🙈 We're not showing anything, but that's by accident (since prevUrl is not provided in the login template, the StepHeaderButtons render nothing)! I will comment out the div that mounts the component in the login template for now…